src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/FirstSquare.tsx

```diff
@@ -72,11 +72,11 @@ const FirstSquare = ({ editPage }: { editPage: boolean }) => {
 
     const response = await Operator.getInstance().getOperators(payload);
     if (response?.pagination?.page > 1) {
-      setOperatorsData([...operatorsData, ...response.operators]);
+      const operators = response.operators.filter((operator: any) => !operatorsData.includes(operator));
+      setOperatorsData([...operatorsData, ...operators]);
```

Comment:
If each operator here is an object it won't work.
Test this code in a console:

```
const operators = [{id: 1, test: 'foo'}];
operators.includes({id: 1, test: 'foo'})
```

Reply:
was added solution with converting to string before include()